metadata/recreation/trails_and_pathways.md

```diff
@@ -0,0 +1,79 @@
+# Title
+
+Utah Trails and Pathways
+
+## ID
+
+a341f781-d092-4079-b0fb-7d4da5eee5ea
+
+## Brief Summary
+
+Polyline dataset of established motorized and non-motorized trails in Utah.
+
+## Summary
+
+Utah Trails and Pathways contains off-road recreational trail features that are primarily used by hikers, pedestrians, and mountain bikers. UGRC developed this dataset using data from a variety of sources including Orbital View, the National Park Service, and the US Forest Service.
+
+## Description
+
+### What is the dataset?
+
+This dataset depicts over 20,000 trail and pathway features used mostly by hikers/pedestrians and bikers. Some Off-Highway Vehicle (OHV) trails may be included. It does not include on-street bike infrastructure, sidewalks, or pedestrian bridges.
```

Comment:
it's worth noting that the on-street infrastructure can be found in the Utah Roads dataset.